nvflare/private/fed/utils/fed_utils.py

```diff
@@ -315,3 +315,15 @@ def get_target_names(targets):
         if name not in target_names:
             target_names.append(t)
     return target_names
+
+
+def get_return_code(process, job_id, workspace):
+    run_dir = os.path.join(workspace, job_id)
+    rc_file = os.path.join(run_dir, FLMetaKey.PROCESS_RC_FILE)
+    if os.path.exists(rc_file):
+        with open(rc_file, "r") as f:
+            return_code = int(f.readline())
+        os.remove(rc_file)
```

Comment:
if somehow the file system has permission error, the os.remove might fail, should we consider that and add some try catch block?

Reply:
Added try catch to handle the return_code read failure.